controllers/articles.controller.js

```diff
@@ -24,7 +28,27 @@ function getArticles(req, res, next) {
         .catch(next);
 };
 
+function patchArticleVotes(req, res, next) {
+    const { article_id } = req.params;
+    const { inc_votes } = req.body;
 
+    if (isNaN(article_id)) {
+        return res.status(400).send({ msg: 'Invalid article_id' });
+    }
+
+    if (typeof inc_votes !== 'number') {
+        return res.status(400).send({ msg: 'Bad request: inc_votes must be a number' });
+    }
+
+    updateArticleVotes(article_id, inc_votes)
+        .then((updatedArticle) => {
+            if (!updatedArticle) {
+                return res.status(404).send({ msg: 'Article not found' });
+            }
```

Comment:
This error handling doesn't need to be here. You already have a Promise.reject in your model for the same thing.